Cargo.toml

```diff
@@ -22,7 +22,7 @@ exclude = [
 too_many_arguments = "allow"
 
 # This crate is particularly sensitive to compiler optimizations
-[profile.dev.package.turbo-tasks-memory]
```

Comment:
`turbo-tasks-memory` isn't compiled in by default, so this `opt-level` isn't useful anymore.